src/Xamarin.Android.Build.Tasks/Utilities/MamJsonParser.cs

```diff
@@ -76,11 +80,11 @@ void ReadClassRewrites (JToken classRewrites)
 				if (methods == null) {
 					continue;
 				}
-				foreach (var method in methods) {
-					var makeStatic = (bool?) method["MakeStatic"] ?? false;
-					var oldName = (string?) method["OriginalName"];
-					var newName = (string?) method["NewName"];
-					var oldSig  = ReadSignature (method["OriginalParams"]);
+				foreach (var method in methods.AsArray()) {
+					var makeStatic = (bool?) method!["MakeStatic"] ?? false;
+					var oldName = (string?) method!["OriginalName"];
+					var newName = (string?) method!["NewName"];
+					var oldSig  = ReadSignature (method!["OriginalParams"]);
```

Comment:
You could do this to get rid of the `!`:
```suggestion
				foreach (var method in methods.AsArray()) {
				        if (method is null)
				            continue;
					var makeStatic = (bool?) method["MakeStatic"] ?? false;
					var oldName = (string?) method["OriginalName"];
					var newName = (string?) method["NewName"];
					var oldSig  = ReadSignature (method["OriginalParams"]);
```
I guess that could happen if the JSON was like `[ null ]`, which is weird for sure.